CONTRIBUTING.md

```diff
@@ -105,3 +105,9 @@ Lowercase is preferred for file and folder names, one exception to this is Files
 - Make sure you cover at least 70% test-case scenario.
 - You can generate "test coverage" for modified files using `yarn test:coverage` command.
 - if you want to test cases for any particular file you can add path along with command for e.g.`yarn test src/Home/Home.test.tsx`
+
+### Environment Variables
+
+- To get`.env` file contact Admin/contributor of this repo.
```

Comment:
Let's recommend that developers create their own firebase projects for local testing instead of everyone using the same `.env` file.